backend/ops_api/ops/utils/auth.py

```diff
@@ -23,20 +23,26 @@ def user_lookup_callback(_jwt_header: dict, jwt_data: dict) -> Optional[User]:
     return User.query.filter_by(id=identity).one_or_none()
 
 
-def get_jwt(key: Optional[str] = None) -> str:
-    jwt_private_key = current_app.config.get("JWT_PRIVATE_KEY") or key
+def create_oauth_jwt(key: Optional[str] = None, header: Optional[str] = None, payload: Optional[str] = None) -> str:
+    """
+    Returns an Access Token JWS from the configured OAuth Client
+    :param key: OPTIONAL - Private Key used for encoding the JWS
+    :param header: OPTIONAL - JWS Header containing algorithm type
+    :param payload: OPTIONAL - Contains the JWS payload
+    :return: JsonWebSignature
+    """
+    jwt_private_key = key or current_app.config.get("JWT_PRIVATE_KEY")
     if not jwt_private_key:
         raise NotImplementedError
 
-    client_id = current_app.config["AUTHLIB_OAUTH_CLIENTS"]["logingov"]["client_id"]
-    payload = {
-        "iss": client_id,
-        "sub": client_id,
+    # client_id = current_app.config["AUTHLIB_OAUTH_CLIENTS"]["logingov"]["client_id"]
+    _payload = payload or {
+        "iss": current_app.config["AUTHLIB_OAUTH_CLIENTS"]["logingov"]["client_id"],
+        "sub": current_app.config["AUTHLIB_OAUTH_CLIENTS"]["logingov"]["client_id"],
         "aud": "https://idp.int.identitysandbox.gov/api/openid_connect/token",
         "jti": str(uuid.uuid4()),
         "exp": int(time.time()) + 300,
```

Comment:
wasn't this defined as an environment variable?

Reply:
we have:

- JWT_ACCESS_TOKEN_EXPIRES
- JWT_REFRESH_TOKEN_EXPIRES
which are both used for our API Token. This is for the `authentication_code` Token sent back to login_gov. We probably should make this a config, or align it with our own `JWT_ACCESS_TOKEN_EXPIRES` - since it should be short-lived anyway.